huaweicloud/services/obs/resource_huaweicloud_obs_bucket.go

```diff
@@ -1456,7 +1456,7 @@ func getObsError(action string, bucket string, err error) error {
 
 // normalize format of storage class
 func normalizeStorageClass(class string) string {
-	var ret string = class
+	var ret = class
```

Comment:
`var ret string = class` is using more in golang.

Reply:
OK, it has been fixed